unix/vncserver/vncsession.man.in

```diff
@@ -16,6 +17,17 @@ appropriate options and starts a window manager on the VNC desktop.
 is rarely called directly and is normally started by the system service
 manager.
 
+.SH -D OPTION
+.B vncsession
+by default forks and detaches so it normally is a systemd Type=forking service.
```

Comment:
I don't think we need to tie things specifically to systemd. It's sufficient to state that it forks and detaches.

Reply:
I removed references to systemd and that shortened this by two lines in the updated PR.

Reply:
Looks nice!